pkgs/by-name/nu/nufetch-for-nixos-module/package.nix

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
There's frankly no reason why this should be in by-name and visible to everyone. The derivation should be located in or near the module since by itself it scantly makes sense. The patch could be entirely eliminated too that way, since then you're just making a wrapper

Reply:
I'm moving this into the module directory, but I'm not sure how you mean I could remove the patch file because while I am wrapping the program I have to patch its source to change the neofetch config directory path inside the bash script